### PR TITLE
Added an ability to use hours, minutes and seconds

### PR DESCRIPTION
**Pull request by savvinsergey**

### CODE REVIEW

File: src/gantt.ts [hunk 1]
```diff
@@ -589,8 +589,8 @@ module powerbi.extensibility.visual {
                             identity: selectionId
                         };
 
-                        task.end = d3.time.day.offset(task.start, task.duration);
-                        task.tooltipInfo = Gantt.getTooltipInfo(task, host.locale, formatters);
+                        task.end = d3.time[settings.general.durationUnit].offset(task.start, task.duration);
```
Comment (ignatvilesov) [inline src/gantt.ts]: What will happen if `settings.general.durationUnit` value isn't in `d3.time`?

File: src/gantt.ts [hunk 2]
```diff
@@ -602,6 +602,45 @@ module powerbi.extensibility.visual {
...
 
         /**
+         * Generate 'Duration' label for tooltip
+         * @param duration The duration of task
+         * @param durationUnit The duration unit for chart
+         */
+        private static generateLabelForDuration(duration: number, durationUnit: string): string {
+            let label: string = "";
+
+            const days = Math.floor(duration / 24);
```
Comment (ignatvilesov) [inline src/gantt.ts]: Please specify a type.

File: src/gantt.ts [hunk 3]
```diff
@@ -602,6 +602,45 @@ module powerbi.extensibility.visual {
...
+            label += days ? `${days} Days ` : ``;
+            if (durationUnit === "day") {
+                return `${duration} Days `;
+            }
+
+            const hours = duration - (days * 24);
```
Comment (ignatvilesov) [inline src/gantt.ts]: Please specify a type.

File: src/gantt.ts [hunk 4]
```diff
@@ -602,6 +602,45 @@ module powerbi.extensibility.visual {
...
+            label += hours ? `${hours} Hours ` : ``;
+            if (durationUnit === "hour") {
+                return duration >= 24
+                    ? label
+                    : `${duration} Hours`;
+            }
+
+            const minutes = duration - ((days * 24) + (hours * 60));
```
Comment (ignatvilesov) [inline src/gantt.ts]: Please specify a type.

File: src/gantt.ts [hunk 5]
```diff
@@ -602,6 +602,45 @@ module powerbi.extensibility.visual {
...
+            label += minutes ? `${minutes} Minutes ` : ``;
+            if (durationUnit === "minute") {
+                return duration >= 60
+                    ? label
+                    : `${duration} Minutes `;
+            }
+
+            const seconds = duration - (days * 24 + hours * 60 + minutes * 60);
```
Comment (ignatvilesov) [inline src/gantt.ts]: Please specify a type.

File: src/gantt.ts (hunk 1 above)
Comment (savvinsergey) [inline src/gantt.ts]: Added verifying

File: src/gantt.ts (hunk 2 above)
Comment (savvinsergey) [inline src/gantt.ts]: Done

File: src/gantt.ts (hunk 3 above)
Comment (savvinsergey) [inline src/gantt.ts]: Done

File: src/gantt.ts (hunk 4 above)
Comment (savvinsergey) [inline src/gantt.ts]: Done

File: src/gantt.ts (hunk 5 above)
Comment (savvinsergey) [inline src/gantt.ts]: Done

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,6 @@
+## 1.1.1
```
Comment (ignatvilesov) [inline CHANGELOG.md]: I suppose that you should use `1.1.0`. 1.1.0 is declared in pbiviz.json and package.json

Comment (ignatvilesov) [inline CHANGELOG.md]: Please also note that the build is broken. There's TSLint error.

Comment (savvinsergey) [inline CHANGELOG.md]: Done

**Pull request merged**